### DISCUSSION TITLE
libMesh::out and libMesh::err

### DISCUSSION THREAD

**Discussion by doquang** [libMesh/libmesh#2982] [Q&A] [answered]
Hi,
Is there any ways to use std::cout/std::cerr instead of libMesh::out/libMesh::err?
Regards,
/ Minh

**Comment by jwpeterson** [comment, marked as answer]
There's nothing stopping you from using std::cout/std::cerr in user code. We also have an option, `--separate-libmeshout` which prevents the redirecting of std::cout and std::cerr that we normally do for parallel runs.
